Tensile/KernelWriterAssembly.py

```diff
@@ -9155,6 +9192,274 @@ def notLocalFullTileElements(self, kernel, edge):
         return self.notLocalFullTileElementsMFMA(kernel, edge)
     else:
         return self.notLocalFullTileElementsVALU(kernel, edge)
+  ##############################################################################
+  # Store Remap: Local Write
+  ##############################################################################
+  def storeRemapAddLocalWrite(self, kernel, ss, addrCalc, sumIdx):
```

Comment:
write offset address do they write MT0 elements contiguously  or write vgprC as  xdlops outputed into LDS and read gather MT0 elements?

i imagine reducing bankconflicts in read critical for global store that depends on read

Reply:
It write vgprC same as xdlops output into LDS.
each thread write vw4 into different lds column.

we have lds padding to prevent the bandconflict.